client/signup/jetpack-connect/new-site/index.jsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import config from 'config';
+import { recordTracksEvent } from 'state/analytics/actions';
+import SiteURLInput from '../site-url-input';
+import ReaderBack from '../../../blocks/reader-full-post/back';
+
+import WordPressLogo from './wordpress-logo';
+import JetpackLogo from './jetpack-logo';
+
+const JetpackNewSite = React.createClass( {
```

Comment:
Since we're now introducing this component, let's use the new ES6 class syntax for it.